components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/GranularityModeSelector.jsx

```diff
@@ -160,18 +160,23 @@ export default class GranularityModeSelector extends React.Component {
     }
 
     getSelectedGranularityLevel() {
-        const { getTimeRangeName, getDateTimeRangeInfo, options } = this.props;
+        const {
+            getTimeRangeName, getDateTimeRangeInfo, getDefaultTimeRange,
+        } = this.props;
```

Comment:
Any reason for the above formatting?

Reply:
This was due to an eslint error. 
This file is remove with PR https://github.com/wso2/analytics-solutions/pull/270